sparkle/src/components/DropdownMenu.tsx

```diff
@@ -8,6 +16,10 @@ import { Icon } from "./Icon";
 import { Item as StandardItem } from "./Item";
 import { Tooltip, TooltipProps } from "./Tooltip";
 
+const ButtonRefContext = React.createContext<MutableRefObject<any> | null>(
```

Comment:
`any` is not allowed :)

Reply:
How would you solve it?
We do want it to take any in that case

Reply:
There's likely a generic type that covers all we need (HTML something or React component smthg)